profiler/options.go

```diff
@@ -257,20 +257,23 @@ func defaultConfig() (*config, error) {
 	if v := os.Getenv("DD_VERSION"); v != "" {
 		WithVersion(v)(&c)
 	}
+
+	tags := make(map[string]string)
 	if v := os.Getenv("DD_TAGS"); v != "" {
-		sep := " "
-		if strings.Index(v, ",") > -1 {
-			// falling back to comma as separator
-			sep = ","
-		}
-		for _, tag := range strings.Split(v, sep) {
-			tag = strings.TrimSpace(tag)
-			if tag == "" {
-				continue
-			}
-			WithTags(tag)(&c)
+		tags = internal.ParseTagString(v)
+		internal.CleanGitMetadataTags(tags)
```

Comment:
I was a little confused by this line. I think that if the user put those tags in `DD_TAGS`, it would mean they wanted to have them there, and we shouldn't/don't need to remove them, right?

Reply:
At the end they will be there. But for tracer will be delivered in optimal way.
And we want to make the processing the same way for both, to simplify the logic.

Reply:
Ah I see. Given the priority order you want (specific env vars, DD_TAGS, then binary) you'll want to have DD_TAGS cleared of the git tags first. This makes more sense to me now. Thanks!